docs/_docs/maintaining/releasing-a-new-version.md

```diff
@@ -7,6 +7,10 @@ title: "Releasing a new version"
 
 The most important thing to understand before making a release is that there's no need to feel nervous. Most things are revertable, and even if you do publish an incomplete gem version, we can always skip that one. Don't hesitate to contact the other maintainers if you feel unsure or don't know what to do next.
 
+## Branches: `master` vs `*-stable`
+
+We maintain several branches for major/minor releases that have since passed. For example, we maintain the `3.9-stable` branch for the `3.9.x` series since this is still used by GitHub Pages. This allows us to more easily make patches for bugs and security vulnerabilities. Maintaining a `*-stable` version is very similar to `master`, except that the jekyllrb.com blog post must (also) land on the `master` branch in order to be published.
```

Comment:
The flow of this line an update:

```suggestion
We maintain several branches for major/minor releases that have since passed. For example, we maintain the `3.9-stable` branch for the `3.9.x` series since this is still used by GitHub Pages. This allows us to make patches for bugs and security vulnerabilities more easily. Maintaining a `*-stable` version is very similar to `master`, except that the blog post must land on the `master` branch in order to be published at jekyllrb.com.
```